modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/rest/resource/releases/SW360Release.java

```diff
@@ -21,6 +22,17 @@
     private String cpeid;
     private String downloadurl;
     private Set<String> mainLicenseIds;
+    private Map<String, String> coordinates;
+    private Set<String> declaredLicense;
+    private Set<String> observedLicense;
+    private String releaseTagUrl;
+    private String softwareHeritageUrl;
```

Comment:
what is the situation of the idea of saving the ID instead of the URL?

Reply:
do you mean for the software Heritage URL?

Reply:
yes, at some point there was the discussion whether to safe the ID of a release instead of a generic URL and I am not sure what the implications here are.